### PR TITLE
Add share_target compat data.

### PR DESCRIPTION
**Pull request by jpmedley**
https://www.chromestatus.com/feature/6124071381106688

### CODE REVIEW

**Comment by jpmedley** [comment]
@tomayac Can you help me figure out what support there is on other browsers?

**Comment by tomayac** [comment]
What you wrote in your [article](https://web.dev/web-share-target/#browser-support) still holds: Chromium >=76 (Android), Chromium >=89 (Chrome OS).

File: html/manifest/share_target.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+              "version_added": "63"
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            },
+            "samsunginternet_android": {
+              "version_added": true
```
Comment (tomayac) [inline html/manifest/share_target.json:40]:
```suggestion
              "version_added": "12"
```

This is where they switched to a Chromium engine >76 (https://developer.samsung.com/internet/release-note.html).

File: html/manifest/share_target.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
+{
+  "html": {
+    "manifest": {
+      "share_target": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/Manifest/share_target",
+          "spec_url": "https://w3c.github.io/manifest/#incubations",
```
Comment (Elchi3) [inline html/manifest/share_target.json:7]: Is this a better spec_url?

```suggestion
          "spec_url": "https://w3c.github.io/web-share-target/#share_target-member",
```

Comment (jpmedley) [reply html/manifest/share_target.json:7]: @tomayac Which spec should we list here?

Comment (tomayac) [reply html/manifest/share_target.json:7]: https://w3c.github.io/web-share-target/#share_target-member is the better one indeed. The incubations spec actually points there, too.

**Review by Elchi3 — APPROVED** [review]

**Pull request merged**